src/ud60x18/Math.sol

```diff
@@ -437,7 +437,19 @@ function pow(UD60x18 x, UD60x18 y) pure returns (UD60x18 result) {
         if (yUint == uUNIT) {
             result = x;
         } else {
-            result = exp2(mul(log2(x), y));
+            if(xUint > uUNIT) {
+                result = exp2(mul(log2(x), y)); 
+            }
+            // 1^y = 1
+            else if(xUint == uUNIT) {
+                result = UNIT;
+            }
+            // x^y = 1/((1/x)^y)
```

Comment:
Can we move this equation in the NatSpec comments, and use [MathJax](https://www.mathjax.org/)?